### PR TITLE
Expand quickstart section (and convert it to adoc)

### PR DESCRIPTION
**Pull request by mbaldessari**

### CODE REVIEW

File: content/learn/quickstart.adoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* Fedora 36 and onwards
+* Debian Bookworm
+* Ubuntu 22.04
+* Mac OSX Big Sur and onwards
+
+=== Red Hat Enterprise Linux 8 and 9
+Make sure that you have both the `appstream` and the `baseos` repositories configured.
+For example on RHEL 8 you will get the following:
+
+[source,sh]
```
Comment (abhatt-rh) [inline content/learn/quickstart.adoc:47]:
```suggestion
[source,terminal]
```

File: content/learn/quickstart.adoc [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+== Patterns quick start
+
+Each pattern can be deployed using the command line. The only requirement is to have 
+`git` and `podman` installed. See the <<installation_prerequisites>> for more information.
+
+Check the `values-\*.yaml` for changes that are needed before deployment. After
+changing the `values-*.yaml` files where needed and pushing them to your git
+repository, you can run `./pattern.sh make install` from your local repository directory and
+that will deploy the datacenter/hub cluster for a pattern. Edge clusters are
+deployed by joining/importing them into ACM on the hub.
```
Comment (abhatt-rh) [inline content/learn/quickstart.adoc:17]: Global comment: Do not hard wrap lines at 80 characters (or at any other length).

Check your editor settings that let you undo hard wrapping on a paragraph.

File: content/learn/quickstart.adoc [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+sudo dnf repolist
+Updating Subscription Management repositories.
+repo id                               repo name
+rhel-8-for-x86_64-appstream-rpms      Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
+rhel-8-for-x86_64-baseos-rpms         Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+----
+
+Install `podman` and `git`:
+
+[source,sh]
```
Comment (abhatt-rh) [inline content/learn/quickstart.adoc:58]:
```suggestion
[source,terminal]
```

File: content/learn/quickstart.adoc [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+changing the `values-*.yaml` files where needed and pushing them to your git
+repository, you can run `./pattern.sh make install` from your local repository directory and
+that will deploy the datacenter/hub cluster for a pattern. Edge clusters are
+deployed by joining/importing them into ACM on the hub.
+
+Alternatively to the `./pattern.sh make install` method, you can use the
+https://operatorhub.io/operator/patterns-operator[validated pattern
+operator] available in the OpenShift console.
+
+For information on using the Validated Patterns Operator please see
```
Comment (abhatt-rh) [inline content/learn/quickstart.adoc:23]: See IBM Style guidance for [please](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#please).
```suggestion
For information on using the Validated Patterns Operator, see
```

File: content/learn/quickstart.adoc [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+link:/infrastructure/using-validated-pattern-operator/[Using the
+Validated Pattern Operator].
+
+Please follow any other post-install instructions for the pattern on
```
Comment (abhatt-rh) [inline content/learn/quickstart.adoc:27]:
```suggestion
Follow any other post-install instructions for the pattern on
```

**Review by abhatt-rh — COMMENTED** [review]
Hi @mbaldessari,
Mostly LGTM. I just added 2 main comments that apply globally :

- Undo hard wrapping on paragraphs 
- Replace `[source,sh]` with `[source,terminal]`

File: content/learn/quickstart.adoc (hunk 2 above)
Comment (mbaldessari) [reply content/learn/quickstart.adoc:17]: Ack, noted!

File: content/learn/quickstart.adoc (hunk 4 above)
Comment (mbaldessari) [reply content/learn/quickstart.adoc:23]: Ack

File: content/learn/quickstart.adoc (hunk 5 above)
Comment (mbaldessari) [reply content/learn/quickstart.adoc:27]: Ack

**Pull request merged**